adapters/AdColony/AdColonyAdapter/GADMediationAdapterAdColony.m

```diff
@@ -24,10 +24,17 @@
 #import "GADMAdapterAdColonyInitializer.h"
 #import "GADMAdapterAdColonyRTBInterstitialRenderer.h"
 #import "GADMAdapterAdColonyRewardedRenderer.h"
+#import "GADMAdapterAdColonyRTBBannerRenderer.h"
 
 static AdColonyAppOptions *GADMAdapterAdColonyAppOptions;
 
 @implementation GADMediationAdapterAdColony {
+  /// Completion handler for signal generation. Returns either signals or an error object.
+  GADRTBSignalCompletionHandler _signalCompletionHandler;
```

Comment:
currently unused. We plan to remove as part of some code/cleanup and formatting.